### ISSUE TITLE
[Web, Canvas] Crash in Skiko with RoundedCornerShape

### ISSUE THREAD

**Issue by DSteve595** [JetBrains/compose-multiplatform#2514]
Drawing a `Surface` with certain `RoundedCornerShape`s crashes Skia. It doesn't crash with `RectangleShape` or `RoundedCornerShape(16.dp)`, only seems to crash when setting radii per corner, e.g. `RounderCornerShape(topStart = 8.dp, topEnd = 8.dp)`.
From the browser console:
```
skiko.wasm:0x266c94 Uncaught RuntimeError: remainder by zero
    at skiko.wasm:0x266c94
    at skiko.wasm:0x2674ce
    at skiko.wasm:0x272f76
    at skiko.wasm:0x2d65f2
    at skiko.wasm:0x34aa97
    at skiko.wasm:0x2c3c7e
    at skiko.wasm:0x2c6aa3
    at skiko.wasm:0x279300
    at skiko.wasm:0x269ade
    at skiko.wasm:0x269522
```

[Sample project](https://github.com/JetBrains/compose-jb/files/10106365/RoundedCornerSkikoCrash.zip)
Run `./gradlew jsBrowserDist && ./gradlew jsBrowserRun`, click the bottom option to repro the crash

**Comment by dima-avdeev-jb** [comment]
Add reproducer to GitHub:
https://github.com/dima-avdeev-jb/reproduce-js-canvas-rounded-corner-shape

**Comment by alexzhirkevich** [comment]
This was fixed in https://github.com/JetBrains/compose-multiplatform-core/pull/550 and fix is available in 1.4.1

**Comment by dima-avdeev-jb** [comment]
Fixed in 1.4.1

**Comment by okushnikov** [comment]
Please check the <a href="https://youtrack.jetbrains.com/issue/CMP-2514">following ticket</a> on YouTrack for follow-ups to this issue. GitHub issues will be closed in the coming weeks.
